### PR TITLE
teams and achievement page

### PR DESCRIPTION
**Pull request by aniruddhvaid1311**
team card and achievement section in about page for company high end individuals.

### CODE REVIEW

**Review by medwut — COMMENTED** [review]
Looks good however there are no images for the team members you added is this a mistake ? 
image

**Comment by uzairzafarr** [comment]
Hi Anirud, 
I noticed that you've used a placeholder as the image source (https://via.placeholder.com/150). 
Could you please update it with the actual images instead?

Thankyou

File: package.json
Comment (s-bonnor) [inline package.json:1]: Why have you created a package.json file?

File: home/templates/pages/feedback 2 (2).html
Comment (s-bonnor) [inline home/templates/pages/feedback 2 (2).html:1]: i don't believe "feedback 2 (2).html" is the correct filename here

File: home/templates/pages/about.html [hunk 1]
```diff
@@ -30,6 +174,130 @@ <h1 class="display-2 mb-4">
...
+            }
+        });
+    </script>
+  
+
+    <!-- Team Grid (Using Flexbox for Responsive Layout) -->
+    <div class="team-grid">
+        <!-- Team Member 1 -->
+        <div class="team-member">
+            <img src="https://via.placeholder.com/150" alt="John Smith" class="team-photo">
```
Comment (s-bonnor) [inline home/templates/pages/about.html:241]: please replace these via.placeholder.com links with in repository images

**Review by s-bonnor — CHANGES_REQUESTED** [review]
external image links need to be replaced
bad filename also needs to be fixed

**Comment by s-bonnor** [comment]
Closed by request

**Pull request closed without merging**